src/app/zap-templates/zcl/data-model/chip/ecosystem-information-cluster.xml

```diff
@@ -0,0 +1,60 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="General"/>
+  <struct name="EcosystemDeviceStruct" isFabricScoped="true" apiMaturity="provisional">
+    <cluster code="0x0750"/>
+    <item fieldId="0" name="DeviceName" type="char_string" optional="true" isFabricSensitive="true" length="64"/>
+    <item fieldId="1" name="DeviceNameLastEdit" type="epoch_us" optional="true" isFabricSensitive="true" default="0"/>
+    <item fieldId="2" name="BridgedEndpoint" type="endpoint_no" optional="true" isFabricSensitive="true"/>
+    <item fieldId="3" name="OriginalEndpoint" type="endpoint_no" optional="true" isFabricSensitive="true"/>
+    <item fieldId="4" name="DeviceTypes" array="true" type="DeviceTypeStruct" isFabricSensitive="true" minLength="1"/>
+    <item fieldId="5" name="UniqueLocationIDs" array="true" type="char_string" isFabricSensitive="true"/>
+    <item fieldId="6" name="UniqueLocationIDsLastEdit" type="epoch_us" isFabricSensitive="true" default="0"/>
+  </struct>
+
+  <struct name="EcosystemLocationStruct" isFabricScoped="true" apiMaturity="provisional">
+    <cluster code="0x0750"/>
+    <item fieldId="0" name="UniqueLocationID" type="char_string" isFabricSensitive="true" length="64"/>
+    <item fieldId="1" name="HomeLocation" type="HomeLocationStruct" optional="true" isFabricSensitive="true"/>
+    <item fieldId="2" name="HomeLocationLastEdit" type="epoch_us" optional="true" isFabricSensitive="true" default="0"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <domain>General</domain>
+    <name>Ecosystem Information</name>
+    <code>0x0750</code>
+    <define>ECOSYSTEM_INFORMATION_CLUSTER</define>
+    <description>Provides extended device information for all the logical devices represented by a Bridged Node.</description>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <!-- cluster revision -->
+    <globalAttribute code="0xFFFD" side="either" value="1"/>
+    <attribute code="0x0000" side="server" define="REMOVED_ON" type="epoch_us" isNullable="true" reportable="true" optional="true">
+      <description>RemovedOn</description>
+      <access op="read" privilege="manage"/>
+    </attribute>
+    <attribute code="0x0001" side="server" define="DEVICE_DIRECTORY" type="array" entryType="EcosystemDeviceStruct" isNullable="true" reportable="true" length="256" minLength="1">
+      <description>DeviceDirectory</description>
+      <access op="read" privilege="manage"/>
+    </attribute>
+    <attribute code="0x0002" side="server" define="LOCATION_DIRECTORY" type="array" entryType="EcosystemLocationStruct" isNullable="true" reportable="true" length="64" minLength="1">
+      <description>LocationDirectory</description>
+      <access op="read" privilege="manage"/>
+    </attribute>
```

Comment:
All three attributes are Fabric Sensitive

Reply:
tl;dr these are fabric scoped not fabric sensitive

When looking at the definitions of fabric-scoped vs fabric-sensitive in the [spec](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Data-Model.adoc#ref_FabricSensitiveQuality) explicitly states "Only event definitions and the fields of a fabric-scoped struct MAY be fabric-sensitive".

If you look at Chris's PR that updates Ecosystem Information Cluster https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9499 this was changed to:
```
| 0x0000  s| RemovedOn             | epoch-us                        |            | N P X   |         | R   M  | O
| 0x0001  s| DeviceDirectory       | list[EcosystemDeviceStruct]     |  1 to 256  | N P X   |         | R M F  | M
| 0x0002  s| LocationDirectory     | list[EcosystemLocationStruct]   |  1 to 64   | N P X   |         | R M F  | M
```

I could find any other xml that had an attribute with `isFabricScoped`, so I used `MonitoringRegistrationStruct` as an example, which is how I got the pattern of:
* Attribute xml entry does not have `isFabricScoped`
* The Struct it is referring to `isFabricScoped`
* The individual members of the Fabric Scoped Struct has Fabric sensitive mebers